performant-software/textlab
# Re-links zonemarks on cut and paste and hand edit

**What this Does**
Re-links the zone markers (facs=#img...) when you cut and paste them into the text field. Also adds a "Relink" button to handle hand-editing the same markers, so you can type the link out if you wish. 

**How to Test**
Go here: https://mel-textlab-pr-191.herokuapp.com/

1. Cut and paste and verify that an entire block works as expected

2. Cut and paste part one link elsewhere in the document

3. Cut and paste the link into a different leaf

4. Try hand-editing the "facs" attribute, then click "Relink" and see if it connects
